docs/runbook.md

```diff
@@ -0,0 +1,245 @@
+Runbook
+=======
+
+For regular operations and maintenance of the Router back-end of the Console
+
+## Once Per New Server
+
+General installation instructions are available on the Helium documentation
+website.
+
+> Start here:
+[Run a Network Server](https://docs.helium.com/use-the-network/run-a-network-server/)
+which goes from `git clone` through to various initial `docker` commands
+such as loading an initial snapshot and testing with a real or virtual device.
+
+Syntax of **Bash** shell is used below.
+
+Create shell aliases.
+
+Router:
+
+```bash
+alias router='docker exec -it helium_router router'
+alias wallet='~/helium-wallet-rs/target/release/helium-wallet'
+```
+
+Console:
+
+```bash
+alias console_down='docker-compose down'
+alias console_psql='docker exec -it helium_postgres psql -U postgres'
+alias console_restart='console_down && console_up'
+alias console_up='docker-compose build && docker-compose up -d && docker-compose logs -f'
+alias console_update='docker-compose build --pull && console_restart'
+```
+
+Miscellaneous:
+
+```bash
+alias alert='notify-send --urgency=low -i "$([ $? = 0 ] && echo terminal || echo error)" "$(history|tail -n1|sed -e '\''s/^\s*[0-9]\+\s*//;s/[;&|]\s*alert$//'\'')"'
+```
+
+With the above set of aliases defined and available in your current shell,
+confirm:
+
+```bash
+router --help
+```
+
+## Stop/Start
+
+On production servers, Console versus Router run on different hosts to
+isolate and distribute workloads.
+
+Because staging has far less traffic, everything runs on one host.
+
+```bash
+docker-compose down
+docker-compose up -d
+```
+
+## Install A Release
+
+Published Docker images are
+[tagged](https://quay.io/repository/team-helium/router?tab=tags)
+as `production` or `staging` as appropriate.
+
+Install:
+
+```bash
+router install production
```

Comment:
What does this do, I've never seen this command?

Reply:
Found this in a bit of bash history back-scroll but obviously haven't tried it.

Anyone else know?

Reply:
Should we keep or remove the section on `router install ...`?
Is it actually used by anyone?

Reply:
Personally never used it

Reply:
So what is the actual release procedure used?

My current understanding is that you build a new docker container, push that to Quay, tag it as "production" via Quay UI (is there an API for that?), and then simply bounce the container via the usual, `docker-compose down && docker-compose up -d`.

Yes?

Reply:
Since nobody seems to use this command, it's been removed from the runbook.